config/tileset/hawkes-bay/imagery/hawkes-bay-2023-2024-0.125m.json

```diff
@@ -1,13 +1,19 @@
 {
   "type": "raster",
   "id": "ts_hawkes-bay-2023-2024-0.125m",
+  "name": "hawkes-bay-2023-2024-0.125m",
```

Comment:
hmm I don't think `name` is in our other individual imagery configs here??

Reply:
not usually. I'll remove.